corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -423,12 +424,24 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         },
 
         selectAllAction: function (e) {
+            var aa = document.getElementsByTagName("input");
+            for (var i = 0; i < aa.length; i++) {
```

Comment:
Instead of iterating over the HTML elements, you ought to be able to iterate over the child views - it's safer than scanning the whole page for checkboxes, and it's more encapsulated. My read of [these docs](https://marionettejs.com/docs/master/marionette.collectionview.html#collectionview-children-iterators-and-collection-functions) is that you can do something like `this.children.each(function(childView) { ... })` and then inside the callback, you can call functions on the child view and/or access its model.